doc/doxygen/stylesheet.css

```diff
@@ -26,3 +26,11 @@ div.doxygen-generated-exception-message {
     padding: 5px;
     font-size: 95%;
 }
+
+div.image img[src="fe_enriched_1d.png"] {
+    width:500px;
```

Comment:
i will keep it as is, unless you have a strong opinion on the image size issue. 

Generally i think it is good to have higher resolution images than the size they are displayed. At the end of the day, that's exactly what CSS is for. 

As for knowing who controls the size, it's easy to find out with [Firebug](https://addons.mozilla.org/en-US/firefox/addon/firebug/) plugin to Firefox. Very handy for web development.